SayTheirNames/Source/Controller/More/MoreView.swift

```diff
@@ -84,119 +143,76 @@ final class MoreView: UIView {
         case .developer:
-            titleLabel.text = Strings.GetInvolvedDev.title
-        case .designer:
-            titleLabel.text = Strings.GetInvolvedDesign.title
+            label.text = Strings.GetInvolved.Developer.title
+        case .twitter:
+            label.text = Strings.GetInvolved.Twitter.title
         }
-        return titleLabel
+        
+        return label
     }
     
-    private func descriptionLabel(_ section: section) -> UILabel {
-        let bodyLabel = UILabel()
-        
-        bodyLabel.translatesAutoresizingMaskIntoConstraints = false
-        bodyLabel.textColor = STNAsset.Color.primaryLabel.color
-        bodyLabel.font = UIFont.STN.ctaBody
-        bodyLabel.numberOfLines = Theme.Components.LineLimit.quintuple
-        
-        switch section {
-        case .history:
-            bodyLabel.text = """
-            At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium
-            voluptatum deleniti atque corrupti quos dolores et quas molestias excepturiasdlnalkd.
-            """
-        case .developer:
-            bodyLabel.text = "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis"
-        case .designer:
-            bodyLabel.text = "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis"
-        }
-        
-        return bodyLabel
+    func getDescriptionLabel(for section: MoreSection, with description: String = "") -> UILabel {
+        let label = UILabel()
+        label.text = description
+        label.font = UIFont.STN.ctaBody
+        label.textColor = STNAsset.Color.primaryLabel.color
+        label.numberOfLines = Theme.Components.LineLimit.none
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
     }
     
-    private func actionButton(_ section: section) -> UIButton {
-        let actionButton = UIButton()
-        actionButton.translatesAutoresizingMaskIntoConstraints = false
-
-        actionButton.titleLabel?.font = UIFont.STN.sectionHeader
-        actionButton.layer.borderWidth = 2
-        actionButton.backgroundColor = STNAsset.Color.actionButton.color
-        actionButton.setTitleColor(STNAsset.Color.actionButtonTint.color, for: .normal)
-        actionButton.heightAnchor.constraint(equalToConstant: 50).isActive = true
+    func getActionButton(for section: MoreSection) -> UIButton {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.titleLabel?.font = UIFont.STN.sectionHeader
+        button.layer.borderWidth = 2
+        button.backgroundColor = STNAsset.Color.actionButton.color
+        button.setTitleColor(STNAsset.Color.actionButtonTint.color, for: .normal)
+        button.heightAnchor.constraint(equalToConstant: 50).isActive = true
+        
+        let tapGesture = STNTapGestureRecognizer(target: self, action: #selector(openLink(sender:)))
```

Comment:
what is the reason for using a gesture recogniser and not a button action? Then it can't be tapped with voice control, there's no highlighting, etc.

Reply:
Good point 👍